cluster/ampagent/cmd/install.go

```diff
@@ -326,43 +327,117 @@ func createInitialNetworks() error {
 func removeInitialNetworks() error {
 	for _, network := range ampnetworks {
 		// Check if network already exists
-		exists, err := Docker.NetworkExists(network)
+		id, err := Docker.NetworkID(network)
 		if err != nil {
 			return err
 		}
-		if !exists {
-			log.Println("Skipping non existing network:", network)
-			return nil
+		if id == "" {
+			// Skipping non existent network
```

Comment:
Prefer comment on the right of the instruction:
`continue // Skipping non existent network`